src/views/CourseList.vue

```diff
@@ -46,15 +46,15 @@ getCourses();
             <table class="table-title">
                 <thead>
                     <tr>
-                        <th class="table-column">Title</th>
-                        <th class="table-column">Description</th>
+                        <th class="table-column">Course Number</th>
+                        <th class="table-column">Name</th>
                         <th class="table-column">Actions</th>
                     </tr>
                 </thead>
                 <tbody>
                     <tr v-for="(item) in courses" :key="item.department">
-                        <td class="table-content">{{ item.department }}</td>
-                        <td class="table-content">{{ item.description }}</td>
+                        <td class="table-content">{{ item.courseNumber }}</td>
```

Comment:
For this, I believe the listing is department + course number
So it would be listed as CMSC 1113.
Since almost all departments have some sort of 1113 class, we should have the second identifier there